lib/views/pages/login_signup/register_form.dart

```diff
@@ -131,18 +133,21 @@ class RegisterFormState extends State<RegisterForm> {
       final String currentUserId = result.data['signUp']['user']['_id'];
       await _pref.saveUserId(currentUserId);
 
-      Navigator.of(context).pushReplacement(
-          MaterialPageRoute(builder: (context) => new JoinOrganization(fromProfile: false,)));
+      Navigator.of(context).pushReplacement(MaterialPageRoute(
+          builder: (context) => new JoinOrganization(
+                fromProfile: false,
+              )));
     }
   }
 
   //get image using camera
+  final _picker = ImagePicker();
   _imgFromCamera() async {
-    File image = await ImagePicker.pickImage(
-        source: ImageSource.camera, imageQuality: 50);
+    PickedFile _image =
```

Comment:
This is only change required to be part of this PR, please remove all unnecessary changes.

Reply:
okay wil close this PR and push from newer branch